content/operate/rc/databases/create-database/create-pro-database-existing.md

```diff
@@ -12,6 +12,10 @@ weight: 15
 
 {{< embed-md "rc-create-db-first-steps.md" >}}
 
+    {{< note >}}
+See [Use case settings](#use-case-settings) to view the default settings for each use case.
+    {{< /note >}}
+
```

Comment:
This has a display glitch on the staging page. I think it's because Markdown treats an indented block as blockquote, but I'm not 100% sure. Same issue with Create New Pro Database, btw.
```suggestion
{{< note >}}See [Use case settings](#use-case-settings) to view the default settings for each use case.{{< /note >}}

```